specification/search/data-plane/Azure.Search/preview/2024-11-01-preview/searchindex.json

```diff
@@ -0,0 +1,3220 @@
+              "values": [
+                {
+                  "value": "none",
+                  "name": "None",
+                  "description": "Do not generate additional query rewrites for this query."
+                },
+                {
+                  "value": "generative",
+                  "name": "Generative",
+                  "description": "Generate alternative query terms to increase the recall of a search request."
+                }
+              ]
+            },
+            "description": "When QueryRewrites is set to `generative`, the query terms are sent to a generate model which will produce 10 (default) rewrites to help increase the recall of the request. The requested count can be configured by appending the pipe character `|` followed by the `count-<number of rewrites>` option, such as `generative|count-3`. Defaults to `None`. This parameter is only valid if the query type is `semantic`.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "semanticQuery",
+            "in": "query",
+            "type": "string",
+            "description": "Allows setting a separate search query that will be solely used for semantic reranking, semantic captions and semantic answers. Is useful for scenarios where there is a need to use different queries between the base retrieval and ranking phase, and the L2 semantic phase.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "debug",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "disabled",
+              "semantic",
+              "vector",
+              "queryRewrites"
+            ],
+            "x-ms-enum": {
+              "name": "QueryDebugMode",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "disabled",
+                  "name": "Disabled",
+                  "description": "No query debugging information will be returned."
+                },
+                {
+                  "value": "semantic",
+                  "name": "Semantic",
+                  "description": "Allows the user to further explore their reranked results."
+                },
+                {
+                  "value": "vector",
+                  "name": "Vector",
+                  "description": "Allows the user to further explore their hybrid and vector query results."
+                },
+                {
+                  "value": "queryRewrites",
+                  "name": "QueryRewrites",
+                  "description": "Allows the user to explore the list of query rewrites generated for their search request."
+                }
+              ]
+            },
+            "description": "Enables a debugging tool that can be used to further explore your search results.",
```

Comment:
Does the debug setting have any special syntax to support debugging a specific set of features? Ex, is this valid, `semantic,vector`? If so, could we extend the description here to mention that.